### ISSUE TITLE
Form controls must have sufficient color contrast against the background

### ISSUE THREAD

**Issue by bmbc-dev-detec** [LocalGovIMS/localgov-ims#328]
User Interface Components, such as input fields, checkboxes, accordion toggles and toggle switches, fail to have sufficient colour contrast against the background. This also includes focus states for elements, such as buttons, which must have a sufficient contrast against the adjacent background that's distinguishable from the default state.

Such components need to be distinguishable by people with moderately low vision. Low contrast controls are more difficult to perceive, and may be completely missed by people with a visual impairment.

**Comment by bmbc-dev-detec** [comment]
Ready for retesting after completion of #416

**Comment by bmbc-dev-detec** [comment]
Closing. Will add any problem found after retesting as new issues
